### DISCUSSION TITLE
Can I Run Sub Process When Build?

### DISCUSSION THREAD

**Discussion by golangaccount** [typst/typst#4008] [General]
### Description

some time ,i want to import a sub process run result into document

example:
in latext, i can use lua to run a application and import the result into the latex document when building.

in typst, i not found the way.

has any plan to support this case?

### Use Case

I don't need to write a shell for advanced processing

**Comment by Enivex** [comment]
Shell escape is never going to be possible, but you have two options

- WASM plugins
- The query system
